src/openrct2/CmdlineSprite.cpp

```diff
@@ -371,7 +374,8 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
             return -1;
         }
 
-        if (!sprite_file_export(spriteIndex, outputPath))
+        rct_g1_element* spriteHeader = &spriteFileEntries[spriteIndex];
```

Comment:
```suggestion
        spriteHeader = &spriteFileEntries[spriteIndex];
```
`spriteHeader` already exists (it is a parameter) so having the variable here again is probably shadowing it.

Reply:
I don't know what you mean. `spriteHeader` is not a parameter of `cmdline_for_sprite()`.

Reply:
The folding fooled me and I thought this was still in `sprite_file_export`